### DISCUSSION TITLE
Notification is not dismissible

### DISCUSSION THREAD

**Discussion by hamzali4642** [gdelataillade/alarm#185] [Q&A]
Hello, I have scheduled an alarm with a custom ring tune and it works perfectly fine. The problem I am facing that I can't dismiss the notification of alarm. It rings the entire ring tune and then it stops as if I set Loop = false. Is there any method to dismiss the notification?

**Comment by gdelataillade** [comment]
Hi @hamzali4642 

Android does not allow me to make this notification dismissible because it is tied to a foreground service (which allows me to trigger the alarm at any time). You would have to tap on the notification to open the app and stop it manually with `Alarm.stop`.
Someday I will add notification action buttons (stop & snooze) #58 so you can interact with the alarm without having to open the app.
